app/bot/bot.go

```diff
@@ -252,11 +252,18 @@ func EscapeMarkDownV1Text(text string) string {
 	return text
 }
 
-// DisplayName returns user's display name or username or id
-func DisplayName(msg Message) string {
-	displayUsername := msg.From.DisplayName
-	if displayUsername == "" {
-		displayUsername = msg.From.Username
+// DisplayName returns user's display name or username or id, if usernameFirst is true it will try to get username first
+// instead of display name
+func DisplayName(msg Message, usernameFirst bool) string {
```

Comment:
this boolean parameter is very confusing. Reading the consumer side with true/false requires IDE help to figure out what this unexpected boolean even means. Probably making it an enum (iota) type with descriptive values will help. Or even better, making two functions with distinct names would be a better approach.